jailer/src/chroot.rs

```diff
@@ -20,46 +20,40 @@ const CURRENT_DIR_NUL_TERMINATED: &[u8] = b".\0";
 pub fn chroot(path: &Path) -> Result<()> {
     // We unshare into a new mount namespace. The call is safe because we're invoking a C library
     // function with valid parameters.
-    if unsafe { libc::unshare(libc::CLONE_NEWNS) } < 0 {
-        return Err(Error::UnshareNewNs(sys_util::Error::last()));
-    }
+    cvt_empty_ok(unsafe { libc::unshare(libc::CLONE_NEWNS) }).map_err(Error::UnshareNewNs)?;
```

Comment:
I find the code on the left-side easier to read. Mostly because it is very clear that you are running a command and clearly checking the return value.

Reply:
I agree to some extend. But the name `cvt_empty_ok` does not read nice and isn't telling the reader of that code what is it actually checking or doing. I guess if we find a better name for it I'll be okay with the change you proposed.